src/internal-links/handler.js

```diff
@@ -106,7 +108,67 @@ export async function internalLinksAuditRunner(auditUrl, context, site) {
   };
 }
 
+async function convertToOpportunity(auditUrl, auditData, context) {
+  const {
+    dataAccess,
+    log,
+  } = context;
+
+  const opportunities = await dataAccess.Opportunity.allBySiteIdAndStatus(auditData.siteId, 'NEW');
+  let opportunity = opportunities.find((oppty) => oppty.getType() === AUDIT_TYPE);
+
+  if (!opportunity) {
+    const opportunityData = {
+      siteId: auditData.siteId,
+      auditId: auditData.id,
+      runbook: 'https://adobe.sharepoint.com/sites/aemsites-engineering/Shared%20Documents/3%20-%20Experience%20Success/SpaceCat/Runbooks/Experience_Success_Studio_Broken_Internal_Links_Runbook.docx?web=1',
+      type: AUDIT_TYPE,
+      origin: 'AUTOMATION',
+      title: 'Broken Internal Links',
+      description: 'Broken internal links can significantly harm website\'s SEO by preventing search engines from properly indexing site, potentially reducing search rankings',
```

Comment:
since, this will shown as a fallback on the UI, can you update it to match the same one as mentioned in the mocks:

> We've detected broken internal links on your website. Broken links can negatively impact user experience and SEO. Please review and fix these links to ensure smooth navigation and accessibility.

Ref:
https://www.figma.com/design/lTwm2PWEC7UBTr7OMFPth7/ESS?node-id=236-129346&node-type=section&t=08vtKUXqpSXz3gVl-0

Reply:
I have updated this description. How do you plan to insert `50` - that's number of links found in this text? Do you need a variable to be replaced there, or you are good with above text?

Reply:
No, good with above text. We have the list and count on the frontend.